src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -287,6 +288,11 @@ export default class AdvancedFilters extends Vue {
             modal={this._chiMajorVersion === 5}
             drag
             closable>
+            <div class="-mb--1">
```

Comment:
For consistency, let's follow the same implementation as we have here:
https://assets.ctl.io/chi/5.6.0/components/popover/?theme=Portal#table-filtering
Looks like we need a `<button>` element instead of `<a>`

Reply:
Also, let's import MB1 utility class from `classes.ts`.